File-Formats/PDF-to-image/Loading-document.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Open PDF file in C# | Syncfusion
+description: This page describes how to open PDF file from stream in C# using Syncfusion PDF to image converter library.
+platform: file-formats
+control: PDF to image
+documentation: UG
+---
+# Open PDF file in C#
+
+## Opening an existing PDF document using constructor
+
+You can open an existing document by using PdfToImageConverter class. The following example shows how to load an existing document from physical path or stream using constructor.
```

Comment:
as we discuss change the description for every topic.

Reply:
Yes I have changed the description of every topic